src/mainboard/sunxi/nezha/Makefile

```diff
@@ -12,6 +12,10 @@ XFEL       ?= xfel
 MEMADDR    ?= 0x40000000
 RELEASE    ?= --release
 
+MODE       ?= release
```

Comment:
this should be in sync with `RELEASE` (see above)

How do you feel about putting this into xtask instead? Currently it's a mix already, and gotten messy.

Reply:
Thanks for the feedback. I'll have a look into xtask and will try to implement that next week

Reply:
xtask at first looks hard, but you will be happy if you go for it.